pkg/gofr/datasource/cassandra/cassandra_test.go

```diff
@@ -82,7 +82,7 @@ func Test_Connect(t *testing.T) {
 
 	cassandraBuckets := []float64{.05, .075, .1, .125, .15, .2, .3, .5, .75, 1, 2, 3, 4, 5, 7.5, 10}
 
-	mockLogger.EXPECT().Logf("connecting to cassandra at %v on port %v to keyspace %v", "host1", 9042, "test_keyspace")
+	mockLogger.EXPECT().Debugf("connecting to cassandra at %v on port %v to keyspace %v", "host1", 9042, "test_keyspace")
```

Comment:
```suggestion
	mockLogger.EXPECT().Debugf("connecting to Cassandra at %v on port %v to keyspace %v", "host1", 9042, "test_keyspace")
```

Maybe everywhere in fact